composer.json

```diff
@@ -79,6 +79,7 @@
         "drupal/jquery_ui_touch_punch": "^1.0",
         "drupal/admin_audit_trail": "^1.0",
         "drupal/block_inactive_users": "^2.0",
+        "drupal/field_permissions": "^1.2",
```

Comment:
this is not needed, it i just normal new fields that getting added in the user form, please check if it works fine without this field level permissions.